src/pages/panel/components/nav/Nav.tsx

```diff
@@ -2,11 +2,25 @@ import alveus from "../../../../assets/alveus.png";
 
 import styles from "./nav.module.scss";
 
-export default function Nav() {
+import IconInfo from "../../../../components/icons/IconInfo";
+
+interface NavProps {
+  onShowWelcomeCardClicked: () => void;
+}
+
+export default function Nav({ onShowWelcomeCardClicked }: NavProps) {
   return (
     <nav className={styles.nav}>
+      <div className={styles.filler} />
```

Comment:
I'm admittedly pretty new to web dev, so I'm not sure if there's a better way to make the info icon right-aligned while maintaining the center justification of the logo and nav title. LMK if there's something better I can do.